helpers/index.ts

```diff
@@ -9,6 +9,18 @@ export const truncateDescription = (text: string, maxWords: number): string => {
     return text;
   };
 
+
+ export const getDate = () => {
```

Comment:
this still needed?

Reply:
getDate needed for a job posting for polybase